docs/config/index.md

```diff
@@ -109,6 +109,12 @@ This might potentially cause some misalignment if a package has different logic
 
 Interpret CJS module's default as named exports.
 
+### alias
+
+- **Type:** `Record<string, string> | Array<{ find: string | RegExp, replacement: string, customResolver?: ResolverFunction | ResolverObject }>`
+
+Define custom aliases when running inside tests. They will be merged with aliases from `resolve.alias`.
```

Comment:
Could it be named something else than `alias` ? 

Genuinely asking cause it's not immediately clear to me how they compare/plug into the vite ones (`alias` in vite 2 and `resolve.alias` in vite 3). Easy to be confused when quickly checking config or looking for an existing issue.

Could it be renamed ? (ie: extraAlias, moduleMapper, testAlias...)

Reply:
It’s on test.alias, what is confusing here?

Reply:
You're absolutely right, no confusion. My bad, I was confused :)